src/components/editor/ResourceComponent.jsx

```diff
@@ -71,10 +73,12 @@ const ResourceComponent = () => {
         <div className="row">
           <div className="col-md-10">
             <h3>
-              {resource.label}
+              <ResourceTitle resource={resource} />
               <CopyToNewButton />
               <PreviewButton />
             </h3>
+            <CopyToNewButton />
+            <PreviewButton />
```

Comment:
So we want these buttons both in the h3 and outside of it?

Reply:
Nope. Same problem as above. Fixed.